ci-operator/jobs/rhpit/interop-tests/rhpit-interop-tests-main-presubmits.yaml

```diff
@@ -936,3 +936,80 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )image-hack-force-propagation-observability-core-automation-release-28,?($|\s.*)
+  - agent: kubernetes
```

Comment:
This is currently a presubmit job is this intended? If it is meant to be a pre-submit job, what is the reason for that?

Reply:
Not sure I understand- this file is automatically generated by 'make update', why should I modify it manually?

Reply:
Pre-submit jobs will run when code is submitted to the source repo that you are specifying in your file path.

So as it stands right now this job will only run when someone submits a PR to the[ interop-tests repo](https://github.com/rhpit/interop-tests) which is just a dummy repo, we use it test things out but there's no real development going on there.

In order for a job to be periodic and run on a cadence we need to use cron in the config file.

Reply:
@ascerra This is a SPIKE scenario, no need to run crons :)